test/web_scraper_test.dart

```diff
@@ -19,10 +18,9 @@ void main() {
       expect(timeElapsed, isNotNull);
     });
     test('Parse tags', () async {
-      productNames = webScraper.getElement(
-          'div.thumbnail > div.caption > h4 > a.title', ['href', 'title']);
+      productNames = webScraper.getElement('div.thumbnail > div.caption > h4 > a.title', ['href', 'title']);
       expect(productNames, isNotNull);
-    }); 
```

Comment:
Please format all dart files using the `flutter format` command.

Reply:
Done :)